### ISSUE TITLE
Refractory metabolic acidosis

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6410]
**Preferred term label:**
Refractory metabolic acidosis

**Synonyms**
Refractory metabolic acidosis

**Definition (free text, please give PubMed ID)**
A refractory metabolic acidosis, that is, of the buildup of acid or depletion of base due to accumulation of metabolic acids.
PMID:  24082513 

**Parent term (use hpo.jax.org/app)**
Metabolic acidosis

**Comment by pnrobinson** [comment]
Please use the clinical modifier for "Refractory" (https://hpo.jax.org/app/browse/term/HP:0031375) plus the term formetabolic acidosis.
We are evolving the HPO to use modifiers rather than having separate terms for clinical modifiers.
